### PR TITLE
railing material appearance inherency

### PR DESCRIPTION
**Pull request by ghost**
sounds like a bug so im putting this to stable

### CODE REVIEW

**Comment by afterthought2** [comment]
Don't think that's the fix. Add code to `on_update_icon` or `update_material`. There's also a hard override of the color on the mapped subtype for some reason.

**Comment by ghost** [comment]
moved to railing file,

image
^^^
everything is fine (color has been inherited, thats aluminium color)
image
steel & glass

File: code/game/objects/structures/railing.dm [hunk 1]
```diff
@@ -57,6 +52,11 @@
 	else
 		..()
 
+/obj/structure/railing/update_materials(keep_health)
+	. = ..()
+	color = material.color
+	alpha = 50 + (material.opacity * 255)
```
Comment (MistakeNot4892) [inline code/game/objects/structures/railing.dm]: `Clamp((50 + material.opacity * 255), 0, 255)`

**Review by MistakeNot4892 — APPROVED** [review]
There should be a material flag you can slap on structures to do this automatically but this will work as a temp fix.

**Pull request merged**